composer.json

```diff
@@ -15,7 +15,8 @@
     "type": "composer-plugin",
     "require": {
         "php": "^7.3",
-        "composer-plugin-api": "^1.1||^2.0"
+        "composer-plugin-api": "^1.1||^2.0",
+        "pestphp/pest": "^0.2"
```

Comment:
Everything sounds good. Just not sure about this on as it may cause circular dependencies. I will check it tonight. Thanks!

Reply:
I think we need the dependency because we are using the `Container` class.

Reply:
Well, yeah - but pest depends on this plugin too. And composer will go crazy with circular dependencies.

I will check tonight. ( or try )